.bluemix/deploy.json

```diff
@@ -19,41 +19,69 @@
             "$ref": "#/messages/deploy.apikeyDescription"
           },
           "type": "string",
-          "pattern": "\\S"
+          "pattern": "\\S",
+          "validator": {
+             "v-type": "api-key"
+          }
       },
      "registry-region": {
         "description": {
           "$ref" : "#/messages/deploy.registryRegionDescription"
         },
         "type": "string",
-        "disabled": false
+        "disabled": false,
+        "validator": {
+            "v-type": "k8s-region",
+            "params": ["api-key"]
```

Comment:
You may want to use the new "cr-region" v-type here ... which restricts the dropdown to container registry regions.